src/rocksdb_options.rs

```diff
@@ -820,6 +821,12 @@ impl CompactOptions {
         }
     }
 
+    pub fn reset_global_manual_compaction_canceled(v: bool) {
```

Comment:
1. "reset" is confusing, maybe just "set"?
2. Now, it is a global switch, instead of keeping it inside `CompactionOptions`, move it to DB seems more appropriate.